test/main.ts

```diff
@@ -266,6 +266,22 @@ test('.json() with 204 response and empty body', async t => {
 	await server.close();
 });
 
+test('.json() with 202 response and empty body', async t => {
```

Comment:
Can you also add tests for 202s with a body, one with valid JSON and one with invalid JSON (e.g. plain text)?